From 44ccfc47d7d1b8b69d9e1c63b1fa2bade7d8083f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 23 Jan 2017 11:44:43 -0800
Subject: [PATCH] PUBAPI-1358 cloudapi's usage of sdc-clients' IMGAPI client is
 working by luck Reviewed by: Julien Gilli <julien.gilli@joyent.com> Approved
 by: Julien Gilli <julien.gilli@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index c9ec288..720149a 100644
--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
         "bunyan": "1.8.1",
         "cueball": "1.1.9",
         "kang": "1.1.0",
-        "sdc-clients": "10.0.0",
+        "sdc-clients": "10.0.4",
         "ufds": "1.2.0",
         "semver": "2.2.1",
         "nodemailer": "0.3.29",
-- 
2.21.0

